.changeset/serious-houses-watch.md

```diff
@@ -0,0 +1,29 @@
+---
+'@backstage/theme': patch
```

Comment:
Should be a minor bump even if there are no usages in the repository.

Reply:
I've corrected this to be minor